src/Symfony/Component/Serializer/Mapping/AttributeMetadata.php

```diff
@@ -113,6 +122,22 @@ public function getSerializedName(): ?string
         return $this->serializedName;
     }
 
+    /**
+     * Sets if this attribute must be ignored or not.
+     */
+    public function setIgnore(bool $ignore)
```

Comment:
those should also be added to the contract `AttributeMetadataInterface` then?

Reply:
We have deprecated this interface IIRC.

Reply:
It isn't deprecated, just internal, so methods added to this interface.